Core/Inc/Wireless/Wireless.h

```diff
@@ -17,8 +17,18 @@
 
 #define MAX_BUF_LENGTH 128
 
-#define WIRELESS_FEEDBACK_CHANNEL -5	// 2.395 GHz
-#define WIRELESS_COMMAND_CHANNEL -15 // 2.385 GHz
+#define WIRELESS_YELLOW_CHANNELS 0
+#define WIRELESS_BLUE_CHANNELS 1
+
+#define WIRELESS_YELLOW_FEEDBACK_CHANNEL -5 // 2.395 GHz
```

Comment:
Yes, The SX1280 does not support those low frequencies. But to be honest I think the frequencies are not properly set in the SX. I think it has to do with the byte order. The best test to see if the frequency is accepted is to try it and see if it goes through its init. If it does, the frequency is fine and will be the same on the robot (for which I did not see a pull request yet).

Reply:
Alright. These frequencies did work for both the robot and the basestation. So, even though these frequencies do work, could you give me 2 alternative frequencies? Can the frequencies be for example +5 and +15 together with the -5 and -15, instead of the -25 and -35?
And I did not mean to make a PR for robots, because those can still be set by programming them. I needed this for the basestation so that multiple basestations can be dynamically selected by robothub.

Reply:
If they work I would leave it like that, no need to change them. To be honest, I can't give you any working frequencies without testing them like you have. Any combination of frequencies are possible if they work individually.